migration/incremental/021-post-restore.sql

```diff
@@ -0,0 +1,72 @@
+
+CREATE FUNCTION public.promscale_post_restore()
```

Comment:
shouldn't we put this in prom_api? Then we can move this definition to idempotent scripts.

Reply:
sure. I put it in public because that's where `timescaledb_pre_restore` and `timescale_post_restore` are, but I'd rather move it so it can live in idempotent

Reply:
resolved in [486d8b5](https://github.com/timescale/promscale_extension/pull/185/commits/486d8b5e16f713ac139bbee875c8c801f1d9868c)